rabbitmq_amqp_python_client/entities.py

```diff
@@ -156,12 +157,35 @@ class StreamOptions:
 
     Attributes:
         _filter_set: Dictionary of stream filter specifications
+
+    Args:
+        offset_specification: Either an OffsetSpecification enum value or
+                                an integer offset
+        filters: List of filter strings to apply to the stream
     """
 
-    def __init__(self):  # type: ignore
+    def __init__(
+        self,
+        offset_specification: Optional[Union[OffsetSpecification, int]] = None,
+        stream_filters: Optional[list[str]] = None,
```

Comment:
since we are already in stream option I would rename `stream_filters` to `filters`. `stream_` is redundant